Controller/Api/CommentController.php

```diff
@@ -51,9 +51,9 @@ public function __construct(CommentManagerInterface $commentManager)
      *  }
      * )
      *
-     * @View(serializerGroups="sonata_api_read", serializerEnableMaxDepthChecks=true)
+     * @REST\View(serializerGroups="sonata_api_read", serializerEnableMaxDepthChecks=true)
```

Comment:
So PHP understands that this REST is `use FOS\RestBundle\Controller\Annotations as Rest;`?

If it works, I am okay with it.